drafts/developer-guide/pages/corteza-duplicate-detection/index.adoc

```diff
@@ -0,0 +1,61 @@
+include::ROOT:partial$variables.adoc[]
+
+= Corteza duplicate detection
```

Comment:
After reading this document, I know what duplicate detection is but I don't know how to use it.

The document talks a lot about all the different bits of the feature (such as detection and matching rules), but I don't know how to configure any of them.

* What web application do I need to go to and how do I access the duplicate detection configuration interface?
* In the module configuration screen, where you configure duplicate detection, I only see two sections -- strict and non-strict fields. What are those about? Should we note these two options and how they differ? Where are the matching and duplicate rules you were talking about?
* In the `Features` section you list down the bits this feature supports but are not clarified; can you elaborate on those, please?

Reply:
Like I said earlier on during i have not finished working this, I am waiting  for the feature to be released